modules/claims_api/lib/bd/bd.rb

```diff
@@ -151,7 +152,6 @@ def determine_birls_file_number(doc_type, auth_headers)
       end
       birls_file_num
     end
-    # rubocop:enable Metrics/ParameterLists
```

Comment:
Please add this line back to where it was

Reply:
This line wasn't in the code at all on this branch when I was doing the fixes earlier today.  So I put it where I thought it needed to be.  Based on your comment, I have now moved it to where it should be (line 154).  Should be good now.